packages/terser/README.md

```diff
@@ -55,5 +75,6 @@ to modify the default behaviour.
 
 ## Credits
 
-This package was originally developed by [https://github.com/TrySound](TrySound) but is not
+The original terser plugin was developed by [TrySound](https://github.com/TrySound) but is not
 maintained anymore.
+The current implementation is developed and maintained by [tada5hi](https://github.com/tada5hi).
```

Comment:
```suggestion
Based on [`rollup-terser-plugin`](https://www.npmjs.com/package/rollup-plugin-terser), moved here with permission.
```

We tend not to tag individual maintainers.

Reply:
That is fine, but than i would rather remove this line. Besides, sharing the logic for 10 lines of source code, the implementation vary and the term "moved" is defintly wrong!